src/os/windows/crashlog_win.cpp

```diff
@@ -239,7 +239,7 @@ static const uint MAX_FRAMES     = 64;
 		char sym_info_raw[sizeof(IMAGEHLP_SYMBOL64) + MAX_SYMBOL_LEN - 1];
 		IMAGEHLP_SYMBOL64 *sym_info = (IMAGEHLP_SYMBOL64*)sym_info_raw;
 		sym_info->SizeOfStruct = sizeof(IMAGEHLP_SYMBOL64);
-		sym_info->MaxNameLength = MAX_SYMBOL_LEN;
+		sym_info->MaxNameLength = MAX_SYMBOL_LEN - 1;
```

Comment:
That was the real issue for me, wrong size here, so `\0` is written outside the buffer.

Reply:
SymGetSymFromAddr64 does not write a `\0` at all, I tested this, see also the linked bug report.

Reply:
Yeah, I just tested and no `\0` added at all (documentation is misleading).
Anyway, I think it might be better to have `char sym_info_raw[sizeof(IMAGEHLP_SYMBOL64) + MAX_SYMBOL_LEN];` and `sym_info->MaxNameLength = MAX_SYMBOL_LEN; // sym_info->Name[1 + MAX_SYMBOL_LEN]`, then `sym_info->Name[MAX_SYMBOL_LEN]` is valid.

Reply:
If the C-array is changed to a C++ `std::array` and initialised with `{}` then setting the NUL terminator later is not necessary.